packages/iTwinUI-react/src/core/DatePicker/DatePicker.tsx

```diff
@@ -23,6 +23,24 @@ const isSameDay = (a: Date | undefined, b: Date | undefined) => {
   );
 };
 
+const isInDateRange = (
+  a: Date | undefined,
+  b: Date | undefined,
+  c: Date | undefined,
+) => {
+  a && a.setHours(0, 0, 0, 0);
```

Comment:
If the dates are being passed by value and I'm only trying to compare the year, month, and day, does it matter if the dates are being changed within the function?

Reply:
You really are not passing it by value because date is an object so you are changing its internals. This can cause random bugs that are hard to catch.

Reply:
Changed